source/scpp/extra/DVMChecks.cpp

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************************
```

Comment:
This file is generated by `cli/gen/main.d` right ?

Reply:
I recorded that this file was created automatically. And I described how to make modifications.

Reply:
Great. So I don't think we should check it in the repository.

Reply:
/Users/mukeunkim/Work/WorkSpace_BPFK/agora-node/agora/source/scpp/extra/DVMChecks_Generated.cpp

Reply:
This is a video capture of running agora-gen.
[agora-gen.mov.zip](https://github.com/bpfkorea/agora/files/4418536/agora-gen.mov.zip)

Reply:
log : https://gist.github.com/MukeunKim/fe9c1b848463b834260e6e5fa2453dab